node_cli/utils/helper.py

```diff
@@ -418,3 +412,8 @@ def get_ssh_port(ssh_service_name='ssh'):
     except OSError:
         logger.exception('Cannot get ssh service port')
         return DEFAULT_SSH_PORT
+
+
+# TODO: Add a more robust check for contract address and see if doesn't conflict with alias.
+def is_contract_address(value: str) -> bool:
+    return len(value) == 42 and value.startswith('0x')
```

Comment:
can replace with `bool(re.fullmatch(r"0x[a-fA-F0-9]{40}", value))` and remove comment

Reply:
```
>>> value = '0x83b7b3A59857188474FF0269cCC3859b4647UMPQ'
>>> len(value) == 42 and value.startswith('0x')
True
>>> bool(re.fullmatch(r"0x[a-fA-F0-9]{40}", value))
False
```